src/storage/Manager.h

```diff
@@ -77,8 +77,10 @@ class Manager final : public plugin::ComponentManager<Component> {
      * Runs an expire operation on all open backends. This is called by the expiration
      * timer and shouldn't be called directly otherwise, since it should only happen on a
      * separate thread.
+     *
+     * @param current_network_time The network time that the expiration started.
```

Comment:
I'd be okay calling this just `t` as for `Dispatch()` instead of `current_network_time` . The current network time is  what you'd most likely pass as an argument, but doesn't need to be.